### DISCUSSION TITLE
Netbox Instalations

### DISCUSSION THREAD

**Discussion by AVariot** [netbox-community/netbox#13355] [Help Wanted!]
Hi ! I'm trying to install Netbox but i got a problem with it.
I got this : 

"Static Media Failure
The following static media file failed to load: [netbox-print.css](http://127.0.0.1:8000/static/netbox-print.css)
Check the following:

manage.py collectstatic was run during the most recent upgrade. This installs the most recent iteration of each static file into the static root path.
The HTTP service (e.g. nginx or Apache) is configured to serve files from the STATIC_ROOT path. Refer to [the installation documentation](https://docs.netbox.dev/en/stable/installation/) for further guidance.
STATIC_URL: /static/
The file netbox-print.css exists in the static root directory and is readable by the HTTP process.
Click [here](http://127.0.0.1:8000/) to attempt loading NetBox again."

I don't why i have this because i was following the tuto here : https://docs.netbox.dev/en/stable/installation/3-netbox/
Thank you for your help

**Comment by candlerb** [comment]
> Hi ! I'm trying to install Netbox but i got a problem with it.

Since you just asked question #13356 about plugin development, does this mean you have successfully completed your Netbox install now?

If not, then I suggest you fix this before trying to develop plugins.

> I don't why i have this because i was following the tuto here : https://docs.netbox.dev/en/stable/installation/3-netbox/

The last step of that page is to [test the application](https://docs.netbox.dev/en/stable/installation/3-netbox/#test-the-application) in development mode - did you do that, and did it work?

If so, your problem is probably something with your [http server](https://docs.netbox.dev/en/stable/installation/5-http-server/). Review that page for more info.  You haven't said whether you're using Apache or Nginx, but it's likely the problem lies there.  It could be a config error or a permissions error.  With RedHat-derived OSes it could also be SELinux.  You may get some clues from your webserver logfile, or from the output of `dmesg`.

**Reply by AVariot** [reply]
Hi no i have again this problem because i need 2 netbox but the second have this problem

**Reply by AVariot** [reply]
The las step did not work

**Reply by AVariot** [reply]
Need I install apach or nginx if its for only localhost ?

**Reply by candlerb** [reply]
If you are using the development server only, then you don't need Apache or Nginx, and it will serve the static files for you (via python).

A production deployment runs Netbox as a gunicorn process, with an Apache or Nginx reverse proxy in front: the proxy serves the static files, so those requests bypass python entirely.

If you point your web browser directly at gunicorn, then you will see the error you showed.

**Reply by AVariot** [reply]
Thank you for your response
